github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -170,6 +170,33 @@ periodics:
       - --template
       - --ceiling=10
       - --confirm
+- name: periodic-project-infra-cleanup-needs-approver-review
+  interval: 1h
+  annotations:
+    testgrid-create-test-group: "false"
+  labels:
+    preset-github-credentials: "true"
+  decorate: true
+  cluster: kubevirt-prow-control-plane
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/commenter:v20240801-a5d9345e59
+      command:
+      - /ko-app/commenter
+      args:
+      - |-
+        --query=org:kubevirt
+          is:closed
```

Comment:
Does it make sense to remove it from PRs that have the approve in place?  These PRs may still be open?

Reply:
This is just the least denominator currently.

I was thinking of this as a first step - I am still having to think about a way to query those PRs that have received an approver review, since those would be the ones that would require label removal. Like i.e. an approver has reviewed but asked for changes, which would still be eligible for removal of the label - but in which case the `approved` label would not be there.

Reply:
I would leave it like this since if only filtering for approved PRs, closed PRs (without approval) would not get that label removed.

Reply:
Hm, but maybe that would make sense also, since the bot might have closed it without approver review - and we would want to track that.

Reply:
I am happy to introduce this initial cleanup and we can refine it more later if we find the need.